index.html

```diff
@@ -8,14 +8,76 @@
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
+    <link rel="stylesheet" href="/home/coder/Documents/GitHub/HTML-CSS-Coursework-Week2/css/style.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <header>
+        <img id="logo" src="img/karma-logo.svg"  alt="karma logo">
+        <nav>
+            <ul>
+                <li id="meetKarma"> <a href="#"> Meet Karma </a> </li>
+                <li> <a href="#"> How it Works </a> </li>
+                <li> <a href="#"> Store </a> </li>
+                <li> <a href="#"> Blog </a> </li>
+                <li> <a href="#"> Help </a> </li>
+                <li> <a href="#"> Login </a> </li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <article class="intro">
+            <p> Introducing Karma </p>
+            <p id="firstText">Bring WiFi with you, everywhere you go.</p>
+            <button id="learnMore"> Learn More </button>             
+        </article>
+       
+        <article class="part-one">           
+            <p>Everyone needs a little Karma</p>
+        </article>
```

Comment:
This probably doesn't need to be a separate article. It's more like the header on the 3 icons below it - so you could change it to a `h2` or `h3` tag, and move it into the next section.

Reply:
Remember, `<p>` is for both text, not for headers/titles.